tests/test_settings.py

```diff
@@ -732,6 +732,178 @@ class Settings(BaseSettings):
     assert s.dict() == {'pika': 'p!±@'}
 
 
+test_default_env_file = """\
+debug_mode=true
+host=localhost
+Port=8000
+"""
+
+test_prod_env_file = """\
+debug_mode=false
+host=https://example.com/services
+"""
+
+
+@pytest.mark.skipif(not dotenv, reason='python-dotenv not installed')
+def test_multiple_env_file_one(tmp_path):
+    base_env = tmp_path / '.env'
+    base_env.write_text(test_default_env_file)
```

Comment:
```suggestion
    (tmp_path / '.env').write_text(test_default_env_file)
```
is arguably easier to read and more succinct, and below.

Reply:
`base_env` variable is used again.
```
        class Config:
            env_file = [prod_env, base_env]
```

Reply:
Ok